---
layout: page
title:  "Abstracts"
date:   2023-10-14 10:00:00
---

(In the order of jamboree presentation)

**Team #1:** BIOMOD QUERÉTARO<br>
**Title:** Micro-sense: detection of cefotaxime-resistant bacteria through a portable microfluidic device<br>
**Abstract:** Antibiotic-resistant bacteria have raised concerns among researchers over the past several
decades regarding the serious public health problem these represent. These microorganisms have hindered the effectiveness of antimicrobial treatments, increasing the difficulty to find an adequate antibiotic to treat infection-causing bacteria successfully. The CTX-M-15 allele, present in plasmidic DNA of cefotaxime-resistant bacteria, has been related to the production of enzymes that confer resistance to beta-lactam antibiotics. Accordingly, the use of these antibiotics to treat infections caused by any CTX-M-15-containing E. coli results ineffective. Identifying the presence of this gene through
molecular biology techniques is crucial to avoid utilizing beta-lactam antibiotics when facing this type of bacteria. Therefore, the design of a microfluidic device that conducts the amplification of CTX-M-15 via loop-mediated isothermal amplification (LAMP) on-field is proposed, representing a cost-effective, portable, reliable, and rapid method to detect
beta-lactam antibiotic-resistant bacteria containing the CTX-M-15 allele. It is expected that, by implementing this device within practical applications, a trustworthy visual result that
verifies or discards the presence of the CTX-M-15 gene could be provided, functioning as solid evidence to facilitate decision-making when addressing antibiotic resistance problematics.


**Team #2:** Cargo Code-Breakers<br>
**Title:** Cargo Sorting DNA-Robot<br>
**Abstract:** DNA nanorobots have the potential to carry out highly specific tasks in molecular factories and can bring us closer to understanding complex cellular trafficking. While existing research has demonstrated a DNA nanorobot sorting cargo on a 2D origami in solution, imaging of the movement of individual robots remains limited. In this BIOMOD project, we use super resolution microscopy to characterize the motion of the robot as it carries out a random walk and sorts cargo. To visualize the robot and the fiduciary markers on DNA origami tracks, we implemented DNA-PAINT, a novel imaging technique utilizing fluorophores to bypass the diffraction limit. Our movie will be critical in the further evolution of molecular robotics, including optimizing the sorting algorithm and scaling up the number of robots with ever increasing complexity to carry out tasks, opening the door to potential applications in nanoelectronics, medicine, and biomolecular factories.


**Team #3:** FUNDNA<br>
**Title:** DNgnostix Illuminating the Future of Alzheimer’s Diagnosis: Rapid Quantitation of GFAP using Programmable DNA <br>
**Abstract:** Alzheimer’s Disease is a debilitating progressive neurodegenerative disease. As therapies become available, patients need to be diagnosed earlier in order to begin treatment before irreversible neurodegeneration sets in. Our research project addresses the need for an early detection tool for Alzheimer’s Disease by identifying key biomarkers from RNAseq databases and designing a rapid DNA-based detection and quantification method for target genes. Glial Fibrillary Acidic Protein (GFAP) has been shown by multiple studies to be upregulated in patients with Alzheimer’s Disease. It has been confirmed to be elevated in both serum and cerebrospinal fluid samples. We analyzed and selected specific nucleotide sequences within the GFAP gene that could be targeted with our DNA-based detection method. We then designed a DNA Displacement Fluorescence Assay that can rapidly calculate GFAP levels. This method could be adapted as an early detection tool that will improve chances of diagnosis and treatment for Alzheimer’s patients. 


**Team #4:** Team NoKo<br>
**Title:** SynthePHERE <br>
**Abstract:** DNA origami is known as an innovative technology that can build various 2D and 3D nanostructures using DNA strands. In conventional procedure, DNA origami is folded in bulk solution such as in a test tube. Here, we present our project “SynthePHERE”, which aims at the folding of DNA origami nanostructures in a small compartment such as in a liposome. As a first step, giant unilamellar vesicles (GUVs) containing scaffold and staple　strands are prepared. The DNA origami nanostructures are folded in GUVs with the temperature control of the solution. Since scaffold and staple strands are inside of a small reaction system of GUVs, faster and efficient DNA origami folding is expected.  Additionally, introduction of staple strands into GUVs through the pores on the membrane surfaces is carried out. A membrane protein streptolysin O(SLO) is used to form pores with ≈ 27 nm diameter on GUV membrane surfaces. Also, SLO can get sealed when calcium ions are added. By closing SLO pores, scaffold and staple strands are expected to be entrapped in GUVs, allowing to fold DNA origami nanostructures inside GUVs. By introducing different staple strands into a small compartment, various DNA nanostructures can be folded as a response. This input-output system would be applied to liposome-type molecular robots that can respond to outer environments, which could be used as drug delivery system.


**Team #5:** Team SeaSon<br>
**Title:** A New Method of DNA Nanoassembly - DNA Dumbbell Hybridization <br>
**Abstract:** As is well known, a long DNA strand serves as the main strand, and many short strands are added to guide the assembly of this long strand to form a predetermined structure.Recently, the OUC-SeaSon team has invented a new assembly technology, which involves designing DNA strands to self connect to form dumbbell rings, which are then hybridized to achieve the purpose of assembly. At the same time, by modifying the sequence, we can change the angle of hybridization between dumbbells, thereby forming other structures.Due to the structure of z-dna produced in the process of dumbbell ring hybridization, we used polyacrylamide gel electrophoresis and domain proteins that can specifically recognize z-dna, such as ADAR1, to verify the success of hybridization.Our project provides a new solution for the assembly of DNA structures, and the assembled structures also provide raw materials for studying z-dna and its specific binding proteins.


**Team #6:** Team Kansai<br>
**Title:** Can shuriken hit a target? <br>
**Abstract:** As of 2018, cancer affects more than 18.1 million people worldwide and kills more than half of those affected.
Cancer is mostly treated with anticancer drugs. Anticancer drugs are prescribed as intravenous infusions, injections, or oral medications, and once inside the body, they travel throughout the body in the bloodstream to attack cancer cells.
Unlike surgery or radiotherapy, anticancer drug therapy is not a local treatment, but acts on the whole body. Therefore, it damages normal tissues and causes side effects.<br>
So, we propose a drug delivery system (DDS) using liposomes and RCA. In this molecule, DNA is coated around a liposome that encapsulates the drug. In cancer cells, there are tumor markers with specific sequences. The reaction between the tumor markers and their complementary DNA strands with them stuck in the liposome membrane causes the coated DNA to peel off at the target site. When the DNA coating is removed, the liposome breaks down and the encapsulated drugs are released.<br>
The mechanism of action of this molecule requires the realization of three steps: <br>

1. RCA on the liposome membrane.
2. Coat DNA around the liposome.
3. Peel off the coated DNA and break the liposome.

In this project, we examined coating around liposomes with DNA formed by RCA and breaking the liposomes through strand displacement reactions. In the future, our research may contribute to the development of new DDS materials for medical applications.


**Team #7:** YOKABIO<br>
**Title:** DNA Switch Pitch  -Vertex-Switcher- <br>
**Abstract:** Geometrical elements of DNA nanostructure are faces, edges, and vertices. Research focusing on the faces and edges has been reported, such as DNA boxes and DNA tweezers. Since no mechanism has been established to change the number and position of vertices, here we propose a new mechanism that controls the shape by swapping the positions of recessed and protruding vertices using the theory of dual polyhedron. The structure consists of a rhombic dodecahedron created using DNA origami wireframe and ssDNA branched from the edges. When Another ssDNA is added as a signal, the corresponding ssDNA reacts and transform the structure. There are three states: a rhombic dodecahedron, a hexahedron, and an octahedron. Therefore, by arranging the structures regularly, it is possible to create crystals whose density can be changed in response to signals, and there is a possibility that this structure is used as a new building material.


**Team #8:** XMU-Bionova<br>
**Title:** Synthesize DNA-protein condensates as reaction crucibles <br>
**Abstract:** The technologies such as DNA origami have made it possible to engineer biological molecules to form defined structures in vitro. However, manufacturing of multiscale-ordered materials in the complex cellular environment remains challenge. Here, through designing the phase separation behaviors of synthetic DNA and protein molecules, our project constructs several ordered DNA-protein condensates in vivo, which are characterized by fluorescent microscopy. Our ribonucleoprotein nanostructures, by turning the "disordered collaboration" state to the "ordered pipeline", enhance the lycopene pathway's flux and final productivity, exampling how the organization of biomolecules could accelerate the life engineering process.


**Team #9:** Team Sendai<br>
**Title:** Novel DNA Valve for Light-inducible Gas Exchange <br>
**Abstract:** Various chemical exchange systems exist in nature and are crucial in regulating important phenomena in biology and geochemistry. Attempts at replicating these using DNA nanotechnology have mainly focused on liquid-based systems. There exists no example of a DNA nanostructure that controls movement of molecules in the gas phase, leaving the potential of molecular robots to interact with different microenvironments unexplored. Taking inspiration from plant stomata, we propose a DNA origami polymer that facilitates light-inducible gas exchange through the membrane of a soap bubble.<br>
The monomer is a square plate with a slit similar to a vending machine's coin slot. To span the variable thickness of the membrane, monomers self-stack through shape complementarity, with the slits forming a central tunnel. The inside of the tunnel is lined with cholesterol to realize the hydrophobic environment required for gas transport. The structure has two distinct states that dictate whether gas molecules can pass or not: closed and open, the transition between which is controlled by DNA. When closed, the DNA strands form hairpins that cover the entirety of the tunnel, preventing gas flow. Due to their photoresponsivity, the hairpins unfurl under UV exposure, opening the whole tunnel for gas exchange.
Expansion of our structure’s functionality may enable nanorobots to perform biomimetic photosynthesis, pH control of various environments, and even detection, capture, and subsequent treatment of harmful gases.


**Team #10:** Tokyo Alliance<br>
**Title:** DNA Mochi Device <br>
**Abstract:** Although self-assembly of DNA origami into larger structures by fractal assembly, etc., has been achieved, there have been no methods of drastic size expansion.
We devised combining DNA hydrogels with DNA origami and developing nanoscale devices which transform into gels, which further aggregate into a larger gel. Then, inspired by the softness of gel, we call it DNA-Mochi-Device.
First, the device is two types of nanoscale DNA origami, each has long single-stranded DNA with a complementary sequence to each other produced by Rolling-Circle-Amplification as scaffolds. Second, a strand displacement reaction of specific input DNAs and staples releases the scaffold, it binds to complementary scaffolds of other devices and transitions to a DNA hydrogel. Finally, these gels aggregate to micrometer-scale gels.
Because of this drastic change and input-specific gelation, it could be used as a device which traps substance locally or creates occlusion structures within microscale flow paths such as capillaries.


**Team #11:** USYD UFOLD<br>
**Title:** DNA Cargo Bays for Nanoplastic Capture <br>
**Abstract:** Nanoplastics are found all around, including in clothes, cosmetics, food, and even in the northern Atlantic air. Consequently, humans inhale and ingest these plastics regularly, where they build up in the body. They are linked to inflammation, oxidative stress, and neurotoxicity in human cells, among others. This results in the challenge of detecting and removing nanoplastics from the human body. The aim of this project is to design a novel DNA structure to eventually detect and capture specific micro- or nanoplastics. These structures, DNA cargo bays, will be produced using the design software cadnano and assembled. Structural verification will be performed using gel analysis, fluorescence testing, and transmission electron microscopy (TEM). The structures will undergo switching between closed and open states, the open structure allowing capture of target molecules inside the cargo bay. This product acts as proof-of-concept for bloodstream nanoplastic detection and capture.


**Team #12:** Nano-JLU<br>
**Title:** Redefining Drug Resistance Strategy in Cancer Treatment: De Novo Design of a Stimuli-Responsive Peptide-Based Block Copolymer Assembly<br>
**Abstract:** Drug resistance poses a persistent challenge in the context of advanced malignant tumor treatment. Within this arena, we report the self-assembly of a peptide-based block copolymer with the unique capability to trigger cell apoptosis across a wide spectrum of cancer cell phenotypes through plasma membrane rupture (PMR) upon stimuli from tumor microenvironment, such as pH and the overexpression of matrix metalloproteinase 2 (MMP2). The block copolymer consists of three distinct components: a hydrophobic poly(tyrosine)-block-poly(histidine) (pTrp-pHis) unit and a hydrophilic polyethylene glycol (PEG-8) segment, intricately linked by a MMP2-sensitive peptide linker (PLGLAG). This resulting block copolymer self-assembles to generate nano-capsules of approximately 100 nm in diameter when introduced into an aqueous solution. The as-prepared nanocapsules maintain structural integrity and exhibit negligible cytotoxicity under normal physiologic condition. However, once the nanocapsules accumulate inside a tumor due to the enhanced permeability and retention (EPR) effect, cleavage of the block-copolymer by MMP2 and protonation of the pTrp-pHis block lead to collapse of the nanocapsules, releasing the cationic pTrp-pHis block into the tumor microenvironment. The released cationic pTrp-pHis blocks egage in stable electrostatic interaction with the negatively charged tumor cell membranes, eventually inducing PMR-mediated tumor cell apoptosis. <br>
Our research unveils an innovative strategy for combating drug resistance in cancer therapy and holds the potential to address broader drug resistance challenges in the treatment of bacterial and fungal infections.


**Team #13:** Team Tokyo Tech<br>
**Title:** Micro Invader Game<br>
**Abstract:** Our love of biology and games led us to note the similarities between the Invader Game and the immune system in terms of defeating foreign enemies. We therefore decided to create an immune-like molecular system at the microscale. We call it the "Micro invader game". The invader game is simplified into three components: bullets, a cannon and enemies. The bullets consist of DNA-modified microbeads that roll and move on a flat surface  with RNAs for a substrate. The cannon fires the bullets by removing the stopper DNA on the surface that restricts the movement of the bullets. The enemies are made of liposomes, which fluoresce and/or self-destruct when hit by the bullet. The design of the enemies mimics the basic mechanisms of living organisms that respond to external stimuli. We believe our system, especially the sense-response mechanism of the enemy's design, could lead to new insights in the development of artificial life and drug delivery systems.


**Team #14:** UBC BIOMOD<br>
**Title:** A BIOMODular Enzyme Delivery Vehicle to Target Biofilms<br>
**Abstract:** Biofilms are layers of bacterial communities that can adhere to one another within a self-produced matrix. They can attach to a variety of surfaces including human tissue, causing severe healthcare and environmental issues. Traditional strategies for combating biofilms include the use of antibiotics and interference of bacterial layer formation. However, removing biofilms using these methods can be challenging due to antibiotic resistance and unexpected pathogenic features arising from interference strategies. To address this issue, we aim to create a modular enzyme delivery vehicle. This structure consists of a DNA templated liposome, conjugated with variable enzymes, referred to as an “enzymosome”. By forming our liposomes around DNA-origami structures, which can be altered to modify their size and shape, we can create a customizable platform. Among the DNA structures developed – a trihedron, pentahedron, and octahedron – all three demonstrated high stability in CanDo©. Future investigations include testing enzyme synergy with liposomes and validation of the platform in vitro. The modularity of the enzymosome can address biofilms present in various environments such as in cystic fibrosis patients, food facilities, and water systems. By changing the cargo type and liposome size, this delivery vehicle provides potential to be used across a wide range of applications. 


